save-cosv/src/main/kotlin/com/saveourtool/save/cosv/service/CosvService.kt

```diff
@@ -118,7 +123,32 @@ class CosvService(
                     doProcess(rawCosvFileId, user, organization)
                 }
         }
-        .updateRating(user, organization)
+        .collectList()
```

Comment:
may be update kdoc about this new logic? also maybe rename `process` to `processSmthAndAddTags`

Reply:
Done